darts/tests/test_map.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import numpy as np
+import pandas as pd
+import logging
+
+from ..timeseries import TimeSeries
+
+class MapTestCase(unittest.TestCase):
```

Comment:
I think the content of this class belongs rather to a method `test_map()` inside `TimeSeriesTestCase` in `test_timeseries.py`.

Reply:
Done.